import main_figure_1 from '../main/Main-Figure-1.png';
import extended_data_figure_4 from '../main/Extended-Data-Figure-4.png';
import extended_data_figure_2 from '../main/Extended-Data-Figure-2.png';
import main_figure_3 from '../main/Main-Figure-3.png';
import main_figure_4 from '../main/Main-Figure-4.png';
import extended_data_figure_1 from '../main/Extended-Data-Figure-1.png';
import main_figure_2 from '../main/Main-Figure-2.png';
import extended_data_figure_3 from '../main/Extended-Data-Figure-3.png';

export const main_extended_figs = [
	{name: "Main Figure 1", ftype: "main-figure", pdf_url: "../main/Main-Figure-1.pdf", imgsrc: main_figure_1, id: "main_figure_1", caption: 'ENCODE Phase 3 data production. (a-c) Human and (d-f) mouse experiments performed during ENCODE Phase III with data released on the ENCODE Portal, sorted by (a, d) type of assay or (b, e) type of biosample. (c) An illustrative human locus shows signals from several data types. (f) The mouse fetal developmental matrix shows the tissues and stages at which epigenetic features and transcriptomes were assayed.'},
	{name: "Main Figure 2", ftype: "main-figure", pdf_url: "../main/Main-Figure-2.pdf", imgsrc: main_figure_2, id: "main_figure_2", caption: 'Overview of the ENCODE Encyclopedia with a Registry of candidate cis-regulatory elements. The Encyclopedia consists of ground-level and integrative-level annotations that use data processed by the uniform processing pipelines. SCREEN integrates all levels of annotations and raw data and allows users to visualize them in the UCSC genome browser.'},
	{name: "Main Figure 3", ftype: "main-figure", pdf_url: "../main/Main-Figure-3.pdf", imgsrc: main_figure_3, id: "main_figure_3", caption: 'Selection and classification of cCREs to build the Registry of candidate cis-regulatory elements. We begin by filtering and clustering DNase peaks to create representative DHSs (rDHSs). We then select those rDHSs with high DNase signal (maximal Z-score or max-Z across all biosamples with data, see Methods) and high signal for at least one other assay (H3K4me3, H3K27ac or CTCF) to be cCREs. In total, we define 926,535 cCREs in human and 339,815 cCREs in mouse. Based on combinations of signal and genomic context, we classify cCREs into one of these groups: PLS, pELS, dELS, DNase-H3K4me3, or CTCF-only, and their counts are indicated (k: thousand; M: million).'},
	{name: "Main Figure 4", ftype: "main-figure", pdf_url: "../main/Main-Figure-4.pdf", imgsrc: main_figure_4, id: "main_figure_4", caption: 'Experimental testing of cCRE activity in transgenic mouse assays and by comparison with public MPRA and SuRE data. a, The rates at which the 151 predicted enhancers (each centered on a cCRE-dELS) showed activity in transient transgenic mouse assays, stratified by their prediction ranks in each tissue. The bottom, darker bars indicate activity was detected in the predicted tissue, and the top, lighter bars indicate activity was detected in other tissues but not the predicted tissue. b, Four predicted enhancers testing active by transgenic mouse assay. Predicted enhancers (tested regions in dashed horizontal lines bracketed by vertical lines) and nearby cCREs (yellow, green, and grey boxes indicate cCREs-dELS, DNase-only cCREs, and low-DNase cCREs in the corresponding tissues, respectively) are depicted alongside DNase signal (green) and H3K27ac signal (yellow) in forebrain (Fb), midbrain (Mb), hindbrain (Hb), limb (Lm), and heart (Ht). Stained embryo images reveal the tissues where each predicted enhancer tested active. The two predicted hindbrain enhancers were active in additional brain regions (mm1444 in hindbrain and midbrain; mm1489 in hindbrain, midbrain, and neural tube). H3K27ac signal profiles across tissues accurately predicted additional observed activity in related tissues. Overall positive testing rates: mm1502, 3/3 embryos; mm1444, 7/9; mm1492, 5/5; mm1489, 5/5.  c, Percentages of regions testing positive or negative for enhancer activity by MPRA in lymphoblastoid cell lines (MPRA-positive, filled bars; MPRA-negative, white bars). The bars from top to bottom indicate all tested regions, only those tested regions overlapping cell type-agnostic cCREs, and only those tested regions overlapping cCREs identified in GM12878 cells, partitioned by cCRE group. d, Percentages of genomic positions tested by the SuRE assay for promoter activity in K562 cells (SuRE-positive, filled bars; SuRE-negative, white bars). The bars from top to bottom indicate all genomic positions (SuRE is a genome-wide assay), positions overlapping cell type-agnostic cCREs, and positions overlapping cCREs identified in K562 cells, partitioned by cCRE group.'},
	{name: "Extended Data Figure 1", ftype: "extended-data-figure", pdf_url: "../main/Extended-Data-Figure-1.pdf", imgsrc: extended_data_figure_1, id: "extended_data_figure_1", caption: 'Classification of human cCREs is largely consistent across biosamples. For the 25 human (a) and 15 mouse (b) biosamples covered by all four core assays, we analyzed how cCRE classification may differ between biosamples. For each cell type-agnostic group of cCREs, the bars indicate their group classification in specific biosamples, colored by group as indicated. Black indicates a switch of the grouping, e.g., from cell type-agnostic PLS to cell type-specific pELS or CTCF-only.  Two example switches of cCRE grouping between different biosamples are shown in c and d. c, EH38E2652345 is a cCRE-dELS that has high DNase, H3K4me3, and H3K27ac signals in bipolar spindle neurons. In contrast, in cell types at earlier stages of neuronal differentiation, such as embryonic stem cells, iPSCs, and neural progenitor cells, this cCRE only has high DNase and H3K4me3 signals, suggesting that in these cell types the cCRE may be a poised enhancer. d, EH382459760 is a cCRE-dELS that has high DNase, H3K27ac, and CTCF signals in H1-hESCs and iPSCs. However, in further differentiated cell types such as neural progenitors and bipolar spindle neurons, the H3K27ac signal decreases while the CTCF signal remains, and accordingly, EH382459760 is classified as a CTCF-only cCRE. In c and d cCREs colors correspond to group classification defined in a and b. Gray cCREs have low DNase signal.'},
	{name: "Extended Data Figure 2", ftype: "extended-data-figure", pdf_url: "../main/Extended-Data-Figure-2.pdf", imgsrc: extended_data_figure_2, id: "extended_data_figure_2", caption: 'General properties of cCREs. a, Distributions of GRCh38 cCRE width in base pairs stratified by group classification b, Average phyloP score in the ± 250 bp from the center of each cCRE stratified by cell type-agnostic cCRE group: PLS (red), pELS (orange), dELS (yellow), DNase-H3K4me3 (pink), and CTCF-only (blue). In gray are 500,000 300-bp control regions randomly selected from mappable regions of the human genome. c, Fractions of human and mouse cCREs with homology in the other species. In black (no homology) are cCREs that do not map to the other genome. In dark blue (homology only) are cCREs that map to the other genome but do not overlap a cCRE in that genome. In light blue (homology & cCRE) are cCREs that map to cCREs in the other genome, which then reciprocally map back to the original genome. d, Transcription factor ChIP-seq signals support the group classification of cCREs. Violin plots show the average Pol II, EP300, and RAD21 ChIP-seq signals for cCREs belonging to each cCRE group, along with values indicating median signal levels. All ChIP-seq data and cCREs are in GM12878 cells. Colors of violins indicate cCRE groups (PLS, red, N=17,119; pELS, orange, N=29,435; dELS, yellow, N=28,594; DNase-H3K4me3, pink, N=7,298; CTCF-only, blue, N=11,355; DNase-only, green, N=9,394; low-DNase, gray, N=823,340). Boxplots inside violins display median and first and third quartiles.'},
	{name: "Extended Data Figure 3", ftype: "extended-data-figure", pdf_url: "../main/Extended-Data-Figure-3.pdf", imgsrc: extended_data_figure_3, id: "extended_data_figure_3", caption: 'Summary of transcription and transcription factor binding at cCREs. a, Scatterplot depicting percent overlap of various groups of cCREs with RAMPAGE peaks in eight biosamples with matching data vs. the median expression level (in RPM) of the overlapping RAMPAGE peaks. b, The vast majority of high-quality ChIP-seq peaks of chromatin-associated proteins (mostly transcription factors) overlap cell type-agnostic cCREs. The median overlap is 90% across all ChIP-seq experiments. c-d, GRO-seq signal in GM12878 averaged over all cCREs-PLS (c, in red) and cCREs-dELS (d, in yellow) in a ± 2 kb window centred on cCRE centers. The GRO-seq signals around cCREs-PLS were grouped by the orientation of their associated genes. The GRO-seq signals around cCREs-dELS were grouped by genomic strands. Genomic background signal, computed as described in Supplemental Methods, is shown by the grey tick marks and was approximately 0.02 for both strands in GM12878. e, Percentages of the transcription start sites of FANTOM CAGE-associated transcripts in the eleven FANTOM-defined categories that overlap cCREs-PLS (red), cCREs-pELS (orange), or cCREs-dELS (yellow). The TSSs of the majority of coding-associated transcripts (protein-coding mRNA and divergent lncRNAs) overlapped a cCRE-PLS, while the TSSs of the majority of eRNA-like non-coding RNAs (short ncRNAs, antisense lncRNAs, intergenic lncRNAs, sense intronic lncRNAs, and sense overlap RNAs) overlapped a cCRE-dELS.'},
	{name: "Extended Data Figure 4", ftype: "extended-data-figure", pdf_url: "../main/Extended-Data-Figure-4.pdf", imgsrc: extended_data_figure_4, id: "extended_data_figure_4", caption: 't-SNE analysis of human (a) and mouse (b) biosamples based on the H3K27ac signals at their cCREs. To investigate the relationship among biosamples and their tissue or cell types of origin, we performed t-SNE based on the H3K27ac signal at the cCREs-dELS (human: 667,599 and mouse: 209,041) across all  biosamples (human: 228 and mouse: 66). a, Human biosamples formed seven main clusters as determined by K-means clustering. Cluster 1 comprises adult brain tissues and embryonic neurospheres. Cluster 2 comprises tissues from the adrenal gland, heart, leg muscle, and muscular samples of the gastrointestinal (GI) system. Cluster 3 also comprises tissue but those without strong muscle components such as kidney, liver, and mucosa of the gastrointestinal system. Cluster 4 comprises hematopoietic cells and immune tissues including the spleen and thymus. Cluster 5 comprises embryonic stem cells, induced pluripotent stem cells and in vitro differentiated cells from these pluripotent cell types. This cluster also includes two outliers, A673 and SK-N-MC cell lines. Cluster 6 comprises a mixture of cell lines and primary cells. Cluster 7 comprises tissues from embryonic structures such as the placenta and chorion. b, The mouse developmental tissue samples formed three large clusters: brain, liver (hepatic plus fetal hematopoietic systems), and other tissues, with related tissues cluster together, and several tissues (e.g., the four brain regions, face, and limb) display a time-course dependent arrangement of the samples.'},
	
]