import supplementary_figure_15_hcr_gwas from '../supplemental/Supplementary-Figure-15-HCR-GWAS.png';
import supplementary_figure_4_loci_ucsc_screenshots from '../supplemental/Supplementary-Figure-4-loci-UCSC-screenshots.png';
import supplementary_figure_10_registry_coverage from '../supplemental/Supplementary-Figure-10-registry coverage.png';
import supplementary_figure_13_groseq from '../supplemental/Supplementary-Figure-13-GROseq.png';
import supplementary_figure_6_transgenic_mouse_embryos from '../supplemental/Supplementary-Figure-6-transgenic-mouse-embryos.png';
import supplementary_figure_5_ccre_tiers_2 from '../supplemental/Supplementary-Figure-5-ccRE-Tiers-2.png';
import supplementary_figure_7_mpra_sure from '../supplemental/Supplementary-Figure-7-MPRA-SuRE.png';
import supplementary_figure_1_pr from '../supplemental/Supplementary-Figure-1-PR.png';
import supplementary_figure_9_encode_progress_tissues_primary_cells from '../supplemental/Supplementary-Figure-9-ENCODE progress Tissues primary cells.png';
import supplementary_figure_8_conservation from '../supplemental/Supplementary-Figure-8-Conservation.png';
import supplementary_figure_16_gwas_ms from '../supplemental/Supplementary-Figure-16-GWAS-MS.png';
import supplementary_figure_17_rbc_gwas from '../supplemental/Supplementary-Figure-17-RBC-GWAS.png';
import supplementary_figure_2_ccre_classification from '../supplemental/Supplementary-Figure-2-ccRE-classification.png';
import supplementary_figure_3_ccre_classification from '../supplemental/Supplementary-Figure-3-ccRE-classification.png';
import supplementary_figure_11_chromhmm from '../supplemental/Supplementary-Figure-11-ChromHMM.png';
import supplementary_figure_18_prostate_cancer_gwas from '../supplemental/Supplementary-Figure-18-Prostate-Cancer-GWAS.png';
import supplementary_figure_21_z_score_normalization from '../supplemental/Supplementary-Figure-21-Z-score-normalization.png';
import supplementary_figure_14_tf_expression from '../supplemental/Supplementary-Figure-14-TF-expression.png';
import supplementary_figure_19_multi_gwas from '../supplemental/Supplementary-Figure-19-Multi-GWAS.png';
import supplementary_figure_12_fantom_density from '../supplemental/Supplementary-Figure-12-FANTOM-density.png';
import supplementary_figure_20_scz from '../supplemental/Supplementary-Figure-20-SCZ.png';

export const supplemental_figs = [
	{name: "Supplementary Figure 1 PR", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-1-PR.pdf", imgsrc: supplementary_figure_1_pr, id: "supplementary_figure_1_pr", caption: "Testing methods for predicting VISTA enhancers and gene expression. a, PR curves for predicting e11.5 midbrain, hindbrain, neural tube, and limb enhancers. Colors indicate the epigenetic features whose peaks were used to anchor the enhancer predictions and whose signals were used to rank the predictions. b, Same as in a. except that all enhancer predictions were anchored on DHSs in the respective tissue. Predictions were still ranked by the signal levels of the respective epigenetic features indicated by colors as in a. Gray lines indicate the performance of the average rank of DNase and H3K27ac signals. c, Scatter plots depict correlation between predicted and measured transcript expression in e11.5 midbrain with the predictions being H3K4me3 peaks ranked by their H3K4me3 signals (Pearson's correlation r = 0.59), H3K4me3 peaks ranked by their DNase signal (r = 0.19), DHSs ranked by their H3K4me3 signals (r = 0.76), and DHSs ranked by their DNase signals (r = 0.45)."},
	{name: "Supplementary Figure 2 ccRE classification", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-2-ccRE-classification.pdf", imgsrc: supplementary_figure_2_ccre_classification, id: "supplementary_figure_2_ccre_classification", caption: "Details of building the Registry of cCREs. a, Selection of human cCREs. We began by clustering DNase peaks that passed quality-control thresholds (false discovery rate < 0.1%, DNase signal > 10th percentile) and selected a representative DHS (rDHS) for each cluster. We discarded the rDHSs that did not overlap consensus DHSs (see Supplementary Methods). Then, we selected rDHSs with high DNase max-Z scores and additionally high max-Z score for at least one other assay (H3K4me3, H3K27ac, or CTCF) and further filtered out those rDHSs for which high DNase and ChIP Z-scores were not from the same biosample with experimental data (see concordancy test in Supplementary Methods). In total, we resulted in 926,535 cCREs in human. b, Classification of cCREs into five cell type-agnostic groups (PLS, pELS, dELS, DNase-H3K4me3, or CTCF-only) based on their states of high or low H3K4me3, H3K27ac or CTCF max-Z scores and genomic context (TSS-overlapping, TSS-proximal, or TSS-distal). c, Percentages of the 3.2 billion mappable nucleotides of the GRCh38 genome occupied by the five groups of cCREs. d, Breakdown of cCREs by tiers (defined in Box 2 and Supplementary Methods). e, Breakdown of the five groups of cCREs by tiers. f-j, Selection and classification of mouse cCREs as for human in a-e."},
	{name: "Supplementary Figure 3 ccRE classification", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-3-ccRE-classification.pdf", imgsrc: supplementary_figure_3_ccre_classification, id: "supplementary_figure_3_ccre_classification", caption: "Classification of cCREs in a particular biosample. a, Classification of seven groups of cCREs (PLS, pELS, dELS, DNase-H3K4me3, CTCF-only, DNase-only, or low-DNase) in GM12878 by their states of high or low H3K4me3, H3K27ac, or CTCF Z-scores in GM12878. b, Number of each group of GM12878 cCREs. c, Full classification scheme for biosamples covered by all four core assays (H3K4me3, H3K27ac, CTCF, and DNase). d, Distribution of each group of cCREs across the 25 human biosamples fully covered by the four assays. e, Partial classification scheme for biosamples with DNase data that lack one or more types of ChIP-seq data. f, In biosamples lacking DNase data, cCREs are not assigned groups but rather classified as whether or not they have high ChIP-seq signals."},
	{name: "Supplementary Figure 4 loci UCSC screenshots", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-4-loci-UCSC-screenshots.pdf", imgsrc: supplementary_figure_4_loci_ucsc_screenshots, id: "supplementary_figure_4_loci_ucsc_screenshots", caption: "UCSC Genome Browser views of cCREs and the underlying DNase and ChIP data. Three loci are shown—a, SPI1 b, NPAS4, and c, HNF4A—which are active in B cells, bipolar spindle neurons, and hepatocytes, respectively, signified by a larger font for the corresponding biosample names. The cCREs classified in each biosample (PLS in red, pELS in orange, dELS in yellow, DNase-H3K4me3 in pink, CTCF-only in blue, and DNase-only in green, and low-DNase in gray) are shown above the signal profiles for the four core assays (DNase-seq in green, H3K4me3 ChIP-seq in red, H3K27ac ChIP-seq in yellow, and CTCF ChIP-seq in blue) in each of the three biosamples. In bipolar neurons, additional ChIP-seq data for NRSF are shown in GM12878 and HepG2 cells (black), suggesting NRSF binding at the DNase-only cCREs in B cells and hepatocytes."},
	{name: "Supplementary Figure 5 ccRE Tiers 2", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-5-ccRE-Tiers-2.pdf", imgsrc: supplementary_figure_5_ccre_tiers_2, id: "supplementary_figure_5_ccre_tiers_2", caption: "Classification of cCREs into Tiers based on biosample support."},
	{name: "Supplementary Figure 6 transgenic mouse embryos", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-6-transgenic-mouse-embryos.pdf", imgsrc: supplementary_figure_6_transgenic_mouse_embryos, id: "supplementary_figure_6_transgenic_mouse_embryos", caption: "Impact of ENCODE Phase III data on the Registry."},
	{name: "Supplementary Figure 7 MPRA SuRE", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-7-MPRA-SuRE.pdf", imgsrc: supplementary_figure_7_mpra_sure, id: "supplementary_figure_7_mpra_sure", caption: "Coverage of the current Registry of cCREs. a-c, Percentages of human H3K4me3, H3K27ac and CTCF ChIP-seq peaks from biosamples without DNase data that are covered by human cCREs. d-f, Scatter plots of average -log(FDR) of peaks vs percent overlap for H3K4me3 (d), H3K27ac (e) and CTCF (f) ChIP-seq peaks. Cell types with peaks that had a lower average -log10(FDR) tended to have a lower percentage of peaks covered by cCREs. g-i, Percentages of mouse H3K4me3 and H3K27ac ChIP-seq peaks from biosamples without DNase data that are covered by mouse cCREs. j-l, Scatter plots as described in (d-f) for mouse datasets."},
	{name: "Supplementary Figure 8 Conservation", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-8-Conservation.pdf", imgsrc: supplementary_figure_8_conservation, id: "supplementary_figure_8_conservation", caption: "Overlap of cCREs with ChromHMM states. a, Percentages of various groups of GM12878 cCREs that overlap ChromHMM states. b, Percentage of GM12878 cCREs-pELS that overlap ChromHMM states ranked by distance from the nearest TSS. Due to ChromHMM's lower spatial resolution, cCREs-PLS that are closest to TSSs overlap promoter ChromHMM states while those farther away overlap enhancer states."},
	{name: "Supplementary Figure 9 ENCODE progress Tissues primary cells", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-9-ENCODE progress Tissues primary cells.pdf", imgsrc: supplementary_figure_9_encode_progress_tissues_primary_cells, id: "supplementary_figure_9_encode_progress_tissues_primary_cells", caption: "Overlap of cCREs with FANTOM Enhancers and the transcription start sites of FANTOM CAGE-associated transcripts. Histograms of the Z-scores of cCREs intersecting FANTOM enhancers (colored) and not intersecting FANTOM enhancers (gray). Z-scores are plotted for a, DNase; b, H3K4me3; c, H3K27ac; d, H3K4me1; and e, Pol II. f, Percentages of the transcription start sites of FANTOM CAGE-associated transcripts in the eleven FANTOM-defined categories that overlap cCREs-PLS (red), cCREs-pELS (orange), or cCREs-dELS (yellow). The TSSs of the majority of coding-associated transcripts (protein coding mRNA and divergent lncRNAs) overlapped a cCRE-PLS, while the TSSs of the majority of eRNA-like non-coding RNAs (short ncRNAs, antisense lncRNAs, intergenic lncRNAs, sense intronic lncRNAs, and sense overlap RNAs) overlapped a cCRE-dELS."},
	{name: "Supplementary Figure 10 registry coverage", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-10-registry coverage.pdf", imgsrc: supplementary_figure_10_registry_coverage, id: "supplementary_figure_10_registry_coverage", caption: "Conservation of human cCREs. a, The percentage of positions of PLS (red), pELS (orange), dELS (yellow), DNase-H3K4me3 (pink) and CTCF-only (blue) cCREs that overlap the GERP++ set of evolutionarily conserved regions binned by their DNase max-Z score. Bins with fewer than 20 cCREs are omitted. b, Average phyloP scores of human cCREs stratified by homology categories defined in Extended Data Fig. 2b. c, Average phyloP scores across human cCREs stratified by cCRE group and presence of a FANTOM CAGE peak. cCREs that overlap CAGE peaks are designated by black lines while cCREs do not overlap CAGE peaks are designated by gray lines."},
	{name: "Supplementary Figure 11 ChromHMM", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-11-ChromHMM.pdf", imgsrc: supplementary_figure_11_chromhmm, id: "supplementary_figure_11_chromhmm", caption: "Comparison of cCREs with the ChIP-peaks of chromatin-associated proteins and RNA-seq data. a, The vast majority of high-quality ChIP-seq peaks of chromatin-associated proteins overlap cell type-agnostic cCREs. The median overlap is 90% across all ChIP-seq experiments. b, The overlap remains high for ChIP-seq peaks with cCREs predicted to be active in the same cell type: median overlap is 78%, 84% and 74% in GM12878, HepG2, and K562 cells, respectively. c, Overlap of cell type-agnostic (fill circles) and cell type-specific cCREs (open circles) with ChIP-seq peaks of chromatin-associated proteins in GM12878 (green), HepG2 (red), and K562 (blue) stratified by the number of bound proteins per peak. d, Histograms depicting the number of ChIP-seq peak summits contained within each cCRE stratified by cCRE classification in HepG2. e, Expression levels of four sets of genes classified by whether their TSSs overlap a PLS, other high-DNase cCRE, low-DNase cCRE, or no cCRE in cardiac muscle cells, hepatocytes, and neural progenitors, respectively. f, Percent of expressed genes (TPM > 1) with TSSs that overlap each cCRE group in cardiac muscle cells, hepatocytes, and neural progenitor cells. g, Percent of the least tissue-specific (dark gray) and the most tissue-specific (light gray) genes with a nearby (within 10 kb) cCRE-pELS (orange) or cCRE-dELS (yellow) across 23 biosamples with matching data. The least tissue-specific genes are more likely to have a nearby pELS (Wilcoxon paired test, p=7.5E-9), while the most tissue-specific genes are more likely to have a nearby dELS (p=1.7E-7) defined in the same biosample."},
	{name: "Supplementary Figure 12 FANTOM density", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-12-FANTOM-density.pdf", imgsrc: supplementary_figure_12_fantom_density, id: "supplementary_figure_12_fantom_density", caption: "Transcription patterns at cCREs. a-b, GRO-seq signal in GM12878 (a, c) and PRO-seq signal in CD4+ T cells (b, d) averaged over all cCREs-PLS (a-b, in red) and cCREs-dELS (c-d, in yellow) in a ± 2 kb window centred on cCRE centers. The GRO-seq and PRO-seq signals around cCREs-PLS were grouped by the orientation of their associated genes. The GRO-seq and PRO-seq signals around cCREs-dELS were grouped by genomic strands. Genomic background signal, computed as described in Supplemental Methods, is shown by the grey tick marks and was approximately 0.02 for both strands in GM12878 and 0.03 for both strands in CD4+ T-cell. e-f, The percentage of cCREs-PLS (e) and cCREs-ELS (f) with GRO-seq or PRO-seq signal > 0 on both strands (solid bars), the plus strand only or the minus strand only (light-color bars), or neither strand (open bars) in both GM12878 and CD4+ T cells (left set of bars in each plot), CD4+ T cells only (the middle set of bars in each plot), and in GM12878 cells only (the right set of bars in each plot)."},
	{name: "Supplementary Figure 13 GROseq", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-13-GROseq.pdf", imgsrc: supplementary_figure_13_groseq, id: "supplementary_figure_13_groseq", caption: "Analyzing differential gene expression and differential cCRE activity across mouse developmental timepoints. a, Comparison between liver e11.5 and P0 gene expression and cCRE activity at the Apoe locus. Green bars indicate differentially expressed genes, and red and yellow dots indicate cCREs-PLS and cCREs-ELS. The widths of the green bars represent gene lengths. The lines beneath the green bars and above the gene names indicate the positions and orientations of the genes—red for plus genomic strand and blue for minus strand. The heights of bars or dots indicate changes—log2(fold change) or difference in Z-score—between the two timepoints. The cCRE-dELS EM10E0842983 that overlaps a hepatic control region (HCR) is outlined in black. b, A genome browser view of the Apoe locus with H3K27ac, DNase, and RNA-seq signals in the liver across all surveyed timepoints. c, Apoe gene expression and EM10E0842983 H3K27ac level increase coordinately during development (Pearson’s correlation r = 0.94).  d, H3K27ac and DNase Z-scores in mouse biosamples at EM10E0842983. e, H3K27ac activity in human biosamples for EH38E1957033 (HCR.1), EH38E1957037 (HCR.2) and EH38E1957038 (HCR.2). Biosamples are grouped by tissue ontology. e, A genome browser view of HCR.1 and HCR.2 in human, their overlapping cCREs, and epigenomic signals in HepG2 cells."},
	{name: "Supplementary Figure 14 TF expression", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-14-TF-expression.pdf", imgsrc: supplementary_figure_14_tf_expression, id: "supplementary_figure_14_tf_expression", caption: "Results of testing cCREs-ELS using the in vivo transgenic mouse assays. Shown are representative staining images of e11.5 transgenic mouse embryos (with darker staining indicating higher enhancer activity) for the predicted enhancers that displayed reproducible activity in the expected tissue (midbrain, hindbrain, or limb). Enhancer predictions were performed using the average rank of H3K27ac and DNase signals in the respective e11.5 tissue, and three tranches of predictions were chosen for testing: Top ranking, Middle ranking (~1500) and Bottom ranking (~3000). The unique identifier below each embryo (mm number) corresponds to the accession of the enhancer in the VISTA enhancer browser (http://enhancer.lbl.gov/)."},
	{name: "Supplementary Figure 15 HCR GWAS", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-15-HCR-GWAS.pdf", imgsrc: supplementary_figure_15_hcr_gwas, id: "supplementary_figure_15_hcr_gwas", caption: "Additional analysis of cCREs tested by transgenic mouse assays and cCREs overlapping regions tested by functional assays MPRA and SuRE. a, P-R curves for the cCRE-anchored regions predicted to function as enhancers in e11.5 midbrain, hindbrain, or limb, respectively. b-d Bar plots of the overlap of regions tested by transgenic enhancer assays in this paper (b), Gorkin et al. (c), and Sethi et al. (d). Bars are separated into groups by the ranks of the overlapping cCREs. The best cCRE rank for the tested e11.5 forebrain regions was 1259; hence, the regions were divided using cCRE rank 3000. Dark colors indicate activity in the predicted tissue, while light colors indicate activity in non-predicted tissues. Fisher’s exact test p-values are shown for comparing activity in the predicted tissue. Due to the small number of regions in the Sethi set (d), the comparisons between the groups are not statistically significant. e. Stacked histogram displaying the average phyloP score across the 151 tested regions colored dark gray if active or white if inactive. If we only select highly conserved regions (average phyloP > 1), we observe significant enrichment for positive regions (p = 0.03, Fisher's Exact Test).   f, Overlap of cCREs with MPRA+ regions tested in GM12878 cells stratified by cCRE group and biosample origin: LCLs (black), lymphoid lineage (gray), other (white). P-values are shown for Fisher's exact tests. g, Scatter plot of the percent of MPRA+ regions compared to the total number of overlapping MPRA tested regions. Each point is a biosample colored by whether it is an LCL (red), from the lymphoid lineage (purple), or other (gray). LCLs and lymphoid samples have a lower percentage of MPRA+ regions because they overlap more tested regions due to ascertainment bias. h, Overlap of cCREs with SuRE peaks from K562 cells stratified by cCRE group and biosample origin: K562 (black), myeloid lineage (gray), other (white). P-values are shown for Fisher's exact tests."},	
	{name: "Supplementary Figure 16 GWAS MS", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-16-GWAS-MS.pdf", imgsrc: supplementary_figure_16_gwas_ms, id: "supplementary_figure_16_gwas_ms", caption: "Annotating GWAS variants using cCREs. a, Users can select a GWAS from a preloaded list of GWAS in SCREEN. For each study, we included all tag SNPs reported in the study and all SNPs in LD with the tag SNPs (r2 > 0.7). b, SCREEN reports the percentage of LD blocks of a GWAS with at least one SNP overlapping a cCRE. c, Biosamples are ranked by enrichment of SNP-overlapping cCREs with high H3K27ac signals. Top cell and tissue types are displayed here for each study. For a GWAS (e.g., multiple sclerosis), the user can narrow the search by selecting a biosample (e.g., GM12878 with a 1.91 fold enrichment) and analyze the overlapping cCREs. d, Three multiple-sclerosis SNPs overlap GM12878 cCREs proximal to a ZMIZ1 TSS, with rs1250567 overlapping a pELS (EH38E1482636) and rs1250568 and rs1250569 overlapping a PLS (EH38E1482633). e, The two cCREs in d (in asterisks) have high DNase, H3K4me3, and H3K27ac signals in GM12878 and overlap the ChIP-seq peaks of the transcription factor ELF1. In particular, rs1250568 overlaps a high-scoring position of the ELF1 motif (in a box), with the reference allele disrupting the motif. f, ChIA-PET data link EH38E1482633 and EH38E1482636 to ZMIZ1 and PPIF in GM12878."},
	{name: "Supplementary Figure 17 RBC GWAS", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-17-RBC-GWAS.pdf", imgsrc: supplementary_figure_17_rbc_gwas, id: "supplementary_figure_17_rbc_gwas", caption: "Using cCREs to annotate functional SNPs related to red blood cell traits a, Overlap of cCREs with MPRA functional variants (MFV+) and non-functional variants (MFV−) associated with red blood cell traits. MFV+ regions overlap cCREs more frequently than MFV- regions (3.1 fold enrichment, p = 2.4E-5), especially K562 cCREs (5.0 fold enrichment, p = 4.3E-4). b, Genome browser view of RBC SNP rs737092, which is downstream of the RBM38 gene. This variant overlaps a human cCRE-ELS (EH38E2124446), which has a high DNase signal (green) and a high H3K27ac signal (yellow) in K562. rs737092 also overlaps a homologous mouse cCRE (EM10E0721638, black) c, Top ten cell and tissue types ranked by Z-score for DNase (left) and H3K27ac (right) at EH38E2124446. Multiplicative indices indicate multiple biosamples.  d, Top ten mouse biosamples ranked by Z-score for DNase (left) and H3K27ac (right) at EM10E0721638."},
	{name: "Supplementary Figure 18 Prostate Cancer GWAS", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-18-Prostate-Cancer-GWAS.pdf", imgsrc: supplementary_figure_18_prostate_cancer_gwas, id: "supplementary_figure_18_prostate_cancer_gwas", caption: "Using cCREs to annotate functional SNPs related to prostate cancer a, Genome browser view of rs2742624, which was previously shown to affect the expression of UPK3A, with epigenomic signals of the overlapping cCRE EH38E2169396 in LNCaP and PC-3 (prostate cancer cell line). b-c, Top ten biosamples ranked by Z-score for (b) DNase and (c) H3K27ac at EH38E2169396. d, Genes linked to EH38E2169396 by eQTLs and Hi-C interactions."},
	{name: "Supplementary Figure 19 Multi GWAS", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-19-Multi-GWAS.pdf", imgsrc: supplementary_figure_19_multi_gwas, id: "supplementary_figure_19_multi_gwas", caption: "Using cCREs to annotate functional SNPs related to liver traits"},
	{name: "Supplementary Figure 20 SCZ", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-20-SCZ.pdf", imgsrc: supplementary_figure_20_scz, id: "supplementary_figure_20_scz", caption: "Interpreting GWAS variants associated with neuropsychiatric disorders using cCREs. a, An intron of the AGAP1 gene contains several SNPs associated with neuropsychiatric phenotypes. This region has high linkage disequilibrium and contains many cCREs with high DNase signals in the fetal brain. One region of particular interest also has high H3K4me3 and H3K27ac signals in neural cells, suggesting that it is an enhancer in neural cells. We further analyzed a segment of this region that contains cCRE EH38E2086160 (dashed line with an asterisk). b, Expression of AGAP1 in human biosamples. AGAP1 is primarily expressed in the adult brain. Expression values were calculated from whole-cell poly-A RNA-seq experiments and are displayed in tags per million (TPM). c, The mouse region in the Agap1 locus that is homologous to the EH38E2086160-containing human region, also indicated by a dashed line, contains three cCREs with high H3K27ac and DNase signals in the fetal mouse forebrain but not the limb. The mouse cCRE EM10E0042440 is homologous to the human cCRE EH38E2086160. d, EM10E0042440’s H3K27ac signals measured as fold-change between ChIP and input are displayed across 12 mouse fetal tissues at 8 developmental timepoints. Tissues without H3K27ac ChIP-seq data are in gray. The maximal height of the signals is 10. e, Expression of Agap1 In mouse. Agap1 is primarily expressed in fetal brain tissues. Expression values were calculated from whole-cell poly-A RNA-seq experiments and are displayed in TPM. f. EM10E0042440’s H3K27ac signal in the forebrain (pink), midbrain (blue), and hindbrain (green) across mouse fetal development. g, Results from transgenic mouse assays testing the human region indicated with a dashed line in a, which is homologous to the region also indicated with a dashed line in c. Images indicate that this region active in mouse e11.5 brain tissues, with two stained mouse embryos shown."},
	{name: "Supplementary Figure 21 Z score normalization", ftype: "Supplemental", pdf_url: "../supplemental/Supplementary-Figure-21-Z-score-normalization.pdf", imgsrc: supplementary_figure_21_z_score_normalization, id: "supplementary_figure_21_z_score_normalization", caption: "Method for normalizing epigenomics signals. a, Distribution of the H3K27ac signals of the rDHSs on chromosome 1 in five biosamples (B cell, colon, GM12878, K562, and liver; shown in different colors). rDHSs with H3K27ac signals higher than 5 (average of 9.3k rDHSs per biosample) are omitted from this histogram. b, Distributions of the log10 of the H3K27ac signals in a. The log(signal) values of the rDHSs in each biosample roughly follow a normal distribution. c, Distribution of the Z-scores corresponding to the log10(signal) values in b, with a Z-score of −10 assigned to zero signal values."},
	]